package.json

```diff
@@ -532,332 +532,412 @@
                 {
                     "command": "azureDatabases.openInPortal",
                     "when": "view == azureDatabasesExplorer && viewItem == azureextensionui.azureSubscription",
-                    "group": "1@2"
+                    "group": "2@1"
```

Comment:
I like having "Open in Portal" and "Refresh" as a part of the same group, like [I did](https://github.com/microsoft/vscode-azureresourcegroups/blob/main/package.json#L144) in the rg extension. I consider the group to be "common commands across multiple nodes/extensions that are simple but not _that_ useful"

Reply:
I agree with Eric.  Plus having a totally separate group for just "Open in Portal" looks a bit clunky to me.